web_server/users/models.py

```diff
@@ -1,12 +1,19 @@
 from django.db import models
 
-# Removing this model, it is found in the authors app now.
-# class User(models.Model):
-#     email = models.CharField(max_length=256)  # used as username?
-#     bio = models.CharField(max_length=256)
-#     host = models.CharField(max_length=256)
-#     first_name = models.CharField(max_length=256)
-#     last_name = models.CharField(max_length=256)
-#     display_name = models.CharField(max_length=256)
-#     url = models.CharField(max_length=256)
-#     github = models.CharField(max_length=256)
\ No newline at end of file
+from uuid import uuid4
+from django.contrib.auth.models import AbstractUser
+
+# Create your models here.
+
+
+class Author(AbstractUser):
+    # Author ID's must be unique, we use privacy respecting uuid4 to get random 128bit ids.
+    id = models.UUIDField(primary_key=True, default=uuid4,
+                          editable=False)
+    uid = models.URLField(primary_key=False, max_length=500)
+
+    bio = models.TextField()
+    host = models.URLField()
+    display_name = models.CharField(max_length=256)
```

Comment:
Why are first and last name not fields in this model anymore?

Reply:
if u could please check the built in users_author model 
```
.schema users_author
CREATE TABLE IF NOT EXISTS "users_author" ("password" varchar(128) NOT NULL, "last_login" datetime NULL, "is_superuser" bool NOT NULL, "username" varchar(150) NOT NULL UNIQUE, "first_name" varchar(30) NOT NULL, "last_name" varchar(150) NOT NULL, "email" varchar(254) NOT NULL, "is_staff" bool NOT NULL, "is_active" bool NOT NULL, "date_joined" datetime NOT NULL, "id" char(32) NOT NULL PRIMARY KEY, "uid" varchar(500) NOT NULL, "bio" text NOT NULL, "host" varchar(200) NOT NULL, "display_name" varchar(256) NOT NULL, "url" varchar(500) NOT NULL, "github" varchar(200) NOT NULL);
sqlite>
```

The first_name and last_name column already defined, here I just added additional column that doesn't come with.